### PR TITLE
Persisting the AWS_STACK_NAME env variable

### PR DESCRIPTION
**Pull request by nathanpeck**
#521 Add missing environment variable persistence.

### CODE REVIEW

**Comment by deltamualpha** [comment]
:+1: had this same issue, was puzzled that #503 didn't fix it.

File: 01-path-basics/101-start-here/scripts/lab-ide-build.sh [hunk 1]
```diff
@@ -53,6 +53,7 @@ export EKS_SECURITY_GROUPS=$(aws cloudformation describe-stacks --stack-name $AW
 export EKS_SERVICE_ROLE=$(aws cloudformation describe-stacks --stack-name $AWS_MASTER_STACK | jq -r '.Stacks[0].Outputs[]|select(.OutputKey=="EksServiceRoleArn")|.OutputValue')
 
 # Persist lab variables
+echo "AWS_STACK_NAME=$AWS_STACK_NAME" >> ~/.bashrc
```
Comment (miketheman) [inline 01-path-basics/101-start-here/scripts/lab-ide-build.sh]: This is a repeat of line 59. My bet is that `master` isn't packaged as part of the release.

**Comment by geremyCohen** [comment]
@nathanpeck can we add to both .bashrc and .bash_login ?  Thats the best way to cover for both interactive and non-interactive?  If so I think we'd need to do it for all the env vars?

**Comment by dalbhanj** [comment]
This is fixed now @nathanpeck. @miketheman was correct. The updated file (lab-ide-build.sh) was not merged to S3 bucket. Closing this PR

**Pull request closed without merging**